scripts/check_rosdep.py

```diff
@@ -57,6 +57,15 @@ def no_trailing_spaces(buf):
     return clean
 
 
+def no_blank_lines(buf):
+    clean = True
+    for i, l in enumerate(buf.split('\n')[:-1]):
```

Comment:
```suggestion
    for i, line in enumerate(buf.split('\n')[:-1]):
```

Reply:
I just followed convention as it is used in one function above and below.
- https://github.com/incebellipipo/rosdistro/blob/89a360c6d17078f1603b70119d16b5ba72dcb99d/scripts/check_rosdep.py#L53
- https://github.com/incebellipipo/rosdistro/blob/89a360c6d17078f1603b70119d16b5ba72dcb99d/scripts/check_rosdep.py#L76

If you think it should change it `l`->`line`, than I probably should change every single variable that indicates a line. It seems fine to me though, also fits with previous convention.

Reply:
Good call, that's a fair argument.  I don't like single-letter variable names in general (and neither does newer versions of flake8), but I'd rather fit into convention here.